packages/ui/src/fields/Select/Input.tsx

```diff
@@ -25,6 +32,7 @@ export type SelectInputProps = {
   readonly description?: StaticDescription
   readonly descriptionProps?: Record<string, unknown>
   readonly errorProps?: Record<string, unknown>
+  readonly field?: MarkOptional<SelectFieldClient, 'type'>
```

Comment:
We could use `SelectFieldClientWithoutType` here if we export it, but no strong preference here

Reply:
I thought about it, but we're already overloading this PR with type exports I was tying to keep it to a minimal. Maybe in the future we can make that change based on need.